bin/boom

```diff
@@ -101,6 +106,11 @@ def start_container(client, cmd, name, volumes=None, network_mode='host'):
 
         return None
 
+def create_server(tf):
```

Comment:
Could you document this function please?

Reply:
Removing it.